creator-node/test/testTrackUploadDir/testTrackSourceFileDir/trackManifest.m3u8

```diff
@@ -0,0 +1,69 @@
+#EXTM3U
```

Comment:
is this m3u8 used anywhere?

Reply:
yeah this is used on lines 162 and 176 of the filemanager test - https://github.com/AudiusProject/audius-protocol/pull/652/files/eb006516846f4de84ae59577d48c1bc00407bc32#diff-62fb34331ad4f3eec702231b19ab12e8R176

Reply:
lmk if its unclear what is happening i can add further comments

Reply:
TODO - add comment clarifying that the contents of these test files are not important, only the structure / name

Reply:
added comment for ^